### ISSUE TITLE
Readme does not match the links in the start menu

### ISSUE THREAD

**Issue by Flexmaen** [rdp/virtual-audio-capture-grabber-device#23]
I wanted to stream the Desktop as Webcam-Stream.

README.txt tells me:
"Use some helper programs provided in 
 Start menu -> Programs -> Screen Capture Recorder -> record desktp[/broadcast/XXX"

The menu does not show such an option:
![screen_capture_menu](https://user-images.githubusercontent.com/6330276/34323373-73daa396-e846-11e7-9134-0ec9f78d085b.png)

**Comment by rdp** [comment]
yeah it's "record or stream..." but I didn't realize windows 10 had updated
things I should shuffle some...

**Comment by riso38** [comment]
Hello, dm seve.services on instagram for really good grabber. trust me.
